CONTRIBUTING.md

```diff
@@ -71,19 +120,9 @@ yarn run build
 
 This will create a production version of the library in `/dist` directory.
 
-## Unit Tests
```

Comment:
Maybe we should write something here for how we only have placeholder unit tests but this is something we are investigating and hope to add in the future. We could encourage anyone with experience or knowledge about unit testing to open an issue for discussion. cc @sharellb

Reply:
I think this is a good idea!